src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.scss

```diff
@@ -61,4 +61,14 @@
             }
         }
     }
+
+    .sw-settings-item .sw-settings-item__icon {
+        img {
+            /* Make the following with and height style important to prevent overwrites from third party styles */
+            max-width: 16px !important;
+            max-height: 16px !important;
+            width: 16px !important;
+            height: 16px !important;
+        }
```

Comment:
Have you tested this with multiple extensions, e.g. when I tried / started to work on this I tested it with these, because they all add settings items:
- SwagMigrationAssistant
- SwagPayPal
- SwagLanguagePack
And can you please add a before + after screenshot to the PR description 🙂 .

Also I think we have to apply these to `svg` elements as well, as it's not always an image element (only the case for the Assistant out of these three I think).

And I would adjust this styling in the corresponding file and not in a parent component. In this case:
https://github.com/shopware/shopware/blob/733babc23b28f25022f2e810536a87bcffd1b89e/src/Administration/Resources/app/administration/src/module/sw-settings/component/sw-settings-item/sw-settings-item.scss#L24-L36

Reply:
And another issue: Sometimes, for example the SVG icons of the other sections are not square. So forcing them to be square can make them look weird. Means we should focus on `max-width` instead of `width` for example

Reply:
i test it with SwagMigrationAssistant and SwagLanguagePack. SVGs always have a with and height of 16px Screenshots i make on Tuesday

Reply:
![image](https://github.com/user-attachments/assets/b55301e5-83e0-412a-887e-90d4f1218d5d)

Reply:
All in all, i think this is the best solution. If there is an icon which is a rectangle instead of a square, the developer should change the icon or image file. If you solve it like @fabianhueske said, it could also be looked weird because the edges are cut and this looks more weird than a pressed ratio. At the end it looks weird.